From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: deepak1556 <hop2deep@gmail.com>
Date: Sat, 6 Jan 2018 18:28:10 +0530
Subject: fix: expose InternalCallbackScope

This commit exposes InternalCallbackScope in order to allow us access to its internal flags.

diff --git a/src/node_internals.h b/src/node_internals.h
index c8952e59a2b071c94eaba41393df5b2aa238318e..ab664208076ea776fcb2223dbc228cacd3eb7e58 100644
--- a/src/node_internals.h
+++ b/src/node_internals.h
@@ -204,7 +204,7 @@ v8::MaybeLocal<v8::Value> InternalMakeCallback(
     v8::Local<v8::Value> argv[],
     async_context asyncContext);
 
-class InternalCallbackScope {
+class NODE_EXTERN InternalCallbackScope {
  public:
   enum Flags {
     kNoFlags = 0,
